libs/utils/src/celix_log_utils.c

```diff
@@ -95,6 +97,7 @@ celix_log_level_e celix_logUtils_logLevelFromStringWithCheck(const char *str, ce
 }
 
 static inline void celix_logUtils_inlinePrintBacktrace(FILE *stream) {
+#ifndef __UCLIBC__
```

Comment:
Maybe it is better to place the while function in a ifdef, else, endif so that the function argument can be marked as unused for the uclibc case.

e.g. 
```
#ifndef __UCLIBC__
static inline void celix_logUtils_inlinePrintBacktrace(FILE *stream) {
   ...
}
#else 
static inline void celix_logUtils_inlinePrintBacktrace(FILE *stream  __attribute__((unused))) {
  //nop
}
#endif
```

Reply:
Fixed.